element_interface/dandi.py

```diff
@@ -36,13 +38,17 @@ def upload_to_dandi(
         working_directory, str(dandiset_id)
     )  # enforce str
 
-    download(
-        f"https://gui-staging.dandiarchive.org/#/dandiset/{dandiset_id}"
-        if staging
-        else f"https://dandiarchive.org/dandiset/{dandiset_id}",
-        output_dir=working_directory,
+    dandiset_url = f"https://gui-staging.dandiarchive.org/#/dandiset/{dandiset_id}" if staging else f"https://dandiarchive.org/dandiset/{dandiset_id}/draft"
```

Comment:
Just to confirm, is the `draft` string appended to the URL required when not using the staging server?

Reply:
`draft` is still needed for production. 
New data upload will always go to the `draft` version on DANDI archive. 
Users need to explicitly "publish" the `draft` to make it a "release" version